frontend/src/containers/ViewDashboardAdmin.tsx

```diff
@@ -109,6 +113,35 @@ function ViewDashboardAdmin() {
     }
   };
 
+  const closeCopyModal = () => {
+    setIsOpenCopyModal(false);
+  };
+
+  const onCopyDashboard = () => {
+    setIsOpenCopyModal(true);
+  };
+
+  const copyDashboard = async () => {
+    closeCopyModal();
+
+    if (dashboard) {
+      await BackendService.copyDashboard(dashboard.id);
+
+      history.push("/admin/dashboards?tab=draft", {
+        alert: {
+          type: "success",
+          message: `${dashboard.name}
+                    ${t("DashboardListing.DashboardSingular")}
```

Comment:
This message also can use some TLC regarding i18n usage of interpolation and plurals.

Reply:
@mazenkh-amazon DashboardListing.